pkg/resources/shared_database.go

```diff
@@ -23,7 +24,7 @@ var sharedDatabaseSchema = map[string]*schema.Schema{
 		Type:        schema.TypeString,
 		Required:    true,
 		ForceNew:    true,
-		Description: "A fully qualified path to a share from which the database will be created. A fully qualified path follows the format of `\"<provider_account>\".\"<share_name>\"`.",
+		Description: "A fully qualified path to a share from which the database will be created. A fully qualified path follows the format of `\"<organization_name>\".\"<account_name>\".\"<database_name>\"`.",
```

Comment:
There was a discussion in the previous PR that we should revert it to promote/guide users to use org_name.account_name account identifier format, but I see an error here because it should be share_name instead of database_name.